docs/source/templates/pairwise_regression.md

```diff
@@ -12,32 +12,58 @@ meta_description: Template for performing pairwise comparison tasks for regressi
 
 If you need a dataset to train a pairwise regression model, use this template to rate pairs of images based on how similar they are. You can also customize this template to rate different properties of different types of data, such as rating whether the sentiment of two text excerpts of movie reviews is similar. 
 
-## Template Preview
-
-Interactively preview this labeling template:
+## Interactive Template Preview
 
 <div id="main-preview"></div>
```

Comment:
please check how previews will work on this page - there are a lot of labeling configs, it might be a mess.

Reply:
this doesn't change the preview, and this one just has one labeling config: 
https://labelstud.io/templates/pairwise_regression.html works today just fine :)